dll/win32/shcore/main.c

```diff
@@ -0,0 +1,2581 @@
+/*
```

Comment:
What I don't like with these, is that now this DLL duplicates code that's in shlwapi (which wine has moved these to shcore); however now we would end up with inferior C code there that would have to be replaced (where it makes sense) with our fixed implementations from shlwapi, that may be (for some) in c++ ...